### PR TITLE
Resolve issue 81 by adding details on figures/tables

### LINKED ISSUE #81: Provide some specific guidance on figure quality, axis labels, etc.
PLoS one has quite significantly detailed information on submissions:

http://journals.plos.org/plosone/s/submission-guidelines#loc-figures-and-tables
Look at, for example, their figure guidelines:
http://journals.plos.org/plosone/s/figures

and tables guidelines
http://journals.plos.org/plosone/s/tables

I'm guessing that they have guidelines this detailed because they discovered it was necessary.  So I wonder to what extent we should try to develop something like this level of detail (we have an example of what might be useful to ask for in the PLoS guidelines!)

### PR DESCRIPTION
**Pull request by davidlmobley**
Provides some more detailed guidance on how to prepare figures and tables/what we require in terms of those.

Resolves #81 .

### CODE REVIEW

File: _author_instructions/01_policies.md [hunk 1]
```diff
@@ -103,6 +103,31 @@ However, it is still important that your references use Title Case for article t
 You are responsible for checking that any special characters appear properly, and otherwise ensuring that your final reference list is correct and well-formatted, including sufficient detail.
 Pay particular attention to formatting of non-journal articles as it is easy to end up omitting key citation information for content like software, poster presentations, preprints or other materials.
 
+### Figures
+
+Our largest concern in the construction and preparation of figures is their quality in the final PDF, so we do not *require* any specific image format for figures.
+Rather, figures should:
+- Appear polished in the final PDF of the article, even if zoomed in a great deal
```
Comment (mrshirts) [inline _author_instructions/01_policies.md]: Possible addition: "We suggest using vector graphics whenever possible for diagrams, that the images are saved at the resolution that they are actually used in the paper, and that images that are bitmap format (such as jpeg/png/giff), are at least 200 dpi." Open for suggestions on the dpi number.

File: _author_instructions/01_policies.md [hunk 2]
```diff
@@ -103,6 +103,31 @@ However, it is still important that your references use Title Case for article t
...
+- Be of sufficiently high quality that they look good printed, in both color and grayscale (which has implications for color choices)
+- Be stylistically consistent with the rest of the paper (ideally using OpenSans as the font, and having similar font sizes/styles to the rest of the paper)
+- Use words, symbols, and units for axis labels, *e.g.* not just a plot of sigma vs. r but words should explain what sigma and r are, and what units they use
+- Use self-contained captions so that the point of the figure can be understood without referring to details in the text
+- Not be so large that the final PDF is a huge file
```
Comment (mrshirts) [inline _author_instructions/01_policies.md]: Define huge file, such as less than 5 MB, though most papers will be closer to 1-2 MB, and some documents  that are inherently graphics heavy may need to be larger.  And I'm open on the exact numbers.

File: _author_instructions/01_policies.md [hunk 3]
```diff
@@ -103,6 +103,31 @@ However, it is still important that your references use Title Case for article t
...
+
+Our specific graphical goals can usually be best met by making as many figures as possible in a vector graphics format such as `.svg`, `.pdf`, `.eps`, or similar, as these can be zoomed to an arbitrary level and maintain high quality, but also maintain modest sizes.
+If you are using bitmapped graphics, we highly discourage the use of `.jpg` or other lossy formats and instead recommend `.tif`, `.png` or similar.
+If you do use such graphics formats, pay particular attention to them to ensure that resolution is sufficient so that they look excellent even if zoomed or printed at high quality, as you can expect more issues with these formats than for vector graphics.
+
```
Comment (mrshirts) [inline _author_instructions/01_policies.md]: Specify a DPI - and that images that are at least 150 dpi?

Also, one thing that helps is that that the images are saved at the resolution that they are actually appear in the paper (reducing the usage of commands that resize the image within `includegraphics`,

File: _author_instructions/01_policies.md (hunk 2 above)
Comment (davidlmobley) [reply _author_instructions/01_policies.md]: I'd say no more than 10 MB (typical e-mail attachment size) if you want to get concrete about it.

File: _author_instructions/01_policies.md (hunk 3 above)
Comment (davidlmobley) [reply _author_instructions/01_policies.md]: 300 DPI when used at the size they will appear in the paper.

Is there a reason it is important that they not rescale? For example, a 4x4 image at 200 DPI is equivalent to a 2x2 image at 400 DPI when rescaled unless I'm missing something. 

I agree that if they, say, make a 300 DPI image that is 1x1 and rescale to 4x4 then they'll be in trouble; perhaps that's what you're trying to say.

**Comment by davidlmobley** [comment]
Fixed the issues raised by Michael; merging. If more is needed it can be done in a separate PR.

**Pull request merged**